lib/jsdom/browser/resource-loader.js

```diff
@@ -40,14 +41,30 @@ function createResourceLoadHandler(element, resourceUrl, document, loadCallback)
   };
 }
 
-function readFile(url, callback) {
-  const filePath = url
-    .replace(/^file:\/\//, "")
-    .replace(/^\/([a-z]):\//i, "$1:/")
-    .replace(/%20/g, " ");
+exports.readFile = function (filePath, callback) {
+  const readableStream = fs.createReadStream(filePath);
```

Comment:
Nit: can remove the next line in favor of `fs.createReadStream(filePath, { encoding: "utf8" })`